crates/extensions/tedge_log_manager/src/tests.rs

```diff
@@ -134,6 +139,24 @@ async fn spawn_log_manager_actor(
     (mqtt, fs, uploader)
 }
 
+#[tokio::test]
+async fn default_plugin_config() {
+    let tempdir = TempTedgeDir::new();
+    let (_mqtt, _fs, _uploader) = spawn_log_manager_actor(tempdir.path()).await;
+    let plugin_config_content =
+        read_to_string(tempdir.path().join("tedge-log-plugin.toml")).unwrap();
+    let plugin_config_toml: Table = from_str(&plugin_config_content).unwrap();
+
+    let agent_logs_path = format!("{}/agent/software-*", tempdir.path().to_string_lossy());
+    let expected_config = toml! {
```

Comment:
Is this test helpful? The expected and actual values are built using a copy of the same code.

Reply:
Same comment as above.